Compile error for `wasm32` target with `simd128` feature enabled

The `main` branch currently does not compile for `wasm32` with `simd128` enabled. Running

```
RUSTFLAGS="-Ctarget-feature=+simd128" cargo check --target=wasm32-unknown-unknown
```

results in many compile errors. Here's one of them:

```
error[E0609]: no field `arr` on type `u16x8_::u16x8`
   --> src/u16x8_.rs:431:34
    |
431 |             if self.arr[7] > rhs.arr[7] { u16::MAX } else { 0 },
    |                                  ^^^ unknown field
    |
    = note: available field is: `simd`
```

--- a/src/u16x8_.rs
+++ b/src/u16x8_.rs
@@ -411,6 +411,8 @@ impl u16x8 {
         let mask = cmp_gt_mask_i16_m128i(a_biased, b_biased);
 
         Self { sse: mask }
+      } else if #[cfg(target_feature="simd128")] {
+        Self { simd: u16x8_gt(self.simd, rhs.simd) }
       } else if #[cfg(all(target_feature = "neon", target_arch = "aarch64"))] {
         unsafe {
           use core::arch::aarch64::*;

